### ISSUE TITLE
Test Push, PR, and Merge into Development

### ISSUE THREAD

**Issue by GillianTrethewey** [chingu-voyages/v42-toucans-team-02#25]
**__Task Description__**
The team will test each person following the git flow process making a small change to the README.md file.

**__Technical Considerations__**
1. make a branch under develop in GitHub, say, testing-branch
2. git clone files onto local machine the first time, or git pull subsequent times
3. git branch -a
4. list won't show branch under develop
5. git checkout development
6. git checkout testing-branch (same name as the one up on GitHub)
7. git branch -a
8. make your changes
9. git add -A
10. git commit -m "asdf"
11. git push
12. PR up on GitHub to merge the branch into develop
13. repeat from number 2

**Comment by GillianTrethewey** [comment]
New branch to do this on is:
testing/readme-changes

Completed by myself. On to the next person.

**Comment by LyleT** [comment]
Completed - passed to Melissa for wrap-up.

**Comment by mmhernandez** [comment]
@GillianTrethewey / @LyleT - I moved through the git branching workflow, and initiated a pull request. Looks like it needs to be reviewed and approved in order to merge (so the test branch I created can be deleted).

Let me know if there's anything I did incorrectly - or if you're able to merge this through for me. Assuming all is well, I'll be able to do this process again to push my (for real) readme changes for this sprint's task.

Thank you!
